elide-core/src/main/java/com/yahoo/elide/parsers/expression/PermissionToFilterExpressionVisitor.java

```diff
@@ -91,19 +94,19 @@ public FilterExpression visitAND(ExpressionParser.ANDContext ctx) {
 
         // (NO_EVA and UserCheck) should return NO_EVA let the upper level know.
         // For example, FilterExpressionCheck Or (NO_EVA and UserCheck) should evaluate to  NO_EVALUATION_EXPRESSION.
-        if (left == NO_EVALUATION_EXPRESSION && right == USER_CHECK_EXPRESSION) {
+        if (left == NO_EVALUATION_EXPRESSION && right == FALSE_USER_CHECK_EXPRESSION) {
```

Comment:
I believe this simplifies now.  If either side is FALSE_USER_CHECK_EXPRESSION, the entire expression will evaluate to false - so we can just return FALSE_USER_CHECK_EXPRESSION here.